packages/core/src/internal/clients/cache/ICacheClient.ts

```diff
@@ -265,4 +266,8 @@ export interface ICacheClient extends IControlClient {
     sortedSetName: string,
     values: string[] | Uint8Array[]
   ): Promise<CacheSortedSetRemoveElement.Response>;
+  itemType(
```

Comment:
In the specs, this API is called `itemGetType`.  If we want to revisit that naming discussion with the folks who participated in the design, we can; otherwise we need to use that name.

If we change that name then we will also need to change the response name to `ItemGetType.Response`, which might help us disambiguate the two names that I mentioned in my previous comment.